Public/Src/Sandbox/Windows/DetoursServices/DetoursHelpers.cpp

```diff
@@ -1025,3 +1025,11 @@ bool EnumerateDirectory(
 
     return true;
 }
+
+bool ExistsAsFile(const wchar_t* path)
+{
+    DWORD dwAttrib = Real_GetFileAttributesW(path);

```

Comment:
>Real_ [](start = 21, length = 5)

The convention is we should not call Real_* or Detoured_* . We just call the function.

Reply:
Will change

---
In reply to: [298778259](https://github.com/microsoft/BuildXL/pull/536#discussion_r298778259) [](ancestors = 298778259)